### PR TITLE
added filtering info to various refcards

### PR DESCRIPTION
**Pull request by AlexicaWright**

### CODE REVIEW

**Review by martin-neotech — APPROVED** [review]
Looks good, I will try to build it tomorrow.

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala [hunk 1]
```diff
@@ -65,6 +65,17 @@ SHOW DATABASE myDatabase
...
+//
+
+SHOW DATABASES
+YIELD name, currentStatus
+WHERE name CONTAINS 'my' AND currentStatus = 'online'
+###
+
+List information about databases, filtered by name and online status and further refined by conditions on these.
+
+
```
Comment (Hunterness) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala]:
```suggestion

```

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala [hunk 2]
```diff
@@ -65,6 +65,17 @@ SHOW DATABASE myDatabase
 
 List information about the database `myDatabase`.
 
+###assertion=show-one
+//
+
+SHOW DATABASES
```
Comment (Hunterness) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala]: Where you added this we get the order:
```
SHOW DATABASES
SHOW DATABASE name
SHOW DATABASES YIELD ...
SHOW DEFAULT DATABASE
```
Not sure I like that, I'd say either move the new one last or second place (so either you list all different `SHOW` commands first and then the `YIELD`/`WHERE` or collect all `SHOW DATABASES` versions together before the other `SHOW` command versions)

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala [hunk 3]
```diff
@@ -65,6 +65,17 @@ SHOW DATABASE myDatabase
...
+YIELD name, currentStatus
+WHERE name CONTAINS 'my' AND currentStatus = 'online'
+###
+
+List information about databases, filtered by name and online status and further refined by conditions on these.
```
Comment (Hunterness) [inline cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala]: Not sure what I think of the use of `filtering` here, I guess we're filtering the columns so it should be fine but my brain still connects it to filtering rows which is what your `refined by` does :P (this goes for all commands not just database)

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala (hunk 2 above)
Comment (Hunterness) [reply cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala]: To follow the order used for `SHOW ROLES` it should be in second place:
```
SHOW DATABASES
SHOW DATABASES YIELD ...
SHOW DATABASE name
SHOW DEFAULT DATABASE
```

**Review by Hunterness — CHANGES_REQUESTED** [review]

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala (hunk 2 above)
Comment (AlexicaWright) [reply cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala]: Updated!

File: cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala (hunk 3 above)
Comment (AlexicaWright) [reply cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala]: I think that for a reader of the manual, the wording will be fine as `filtering` is not per definition associated with `columns`.

**Review by Hunterness — APPROVED** [review]

**Pull request merged**